sources/common/toddl.go

```diff
@@ -131,6 +131,10 @@ func (ss *SchemaToSpannerImpl) SchemaToSpannerDDLHelper(conv *internal.Conv, tod
 			NotNull: isNotNull,
 			Comment: "From: " + quoteIfNeeded(srcCol.Name) + " " + srcCol.Type.Print(),
 			Id:      srcColId,
+			AutoGen: ddl.AutoGenCol{
+				Name: "None",
```

Comment:
I see "None" being used as a string everywhere. Any reason to not use the None literal ?

Reply:
Because this will be sent to the UI and then it will get converted to an empty string if it is None Literal